apps/condo/lang/en/en.json

```diff
@@ -1700,8 +1700,10 @@
   "notification.messages.NEWS_ITEM_EMERGENCY_MESSAGE_TYPE.push.title": "{title}",
   "notification.messages.B2C_APP_MESSAGE_PUSH.push.title": "{title}",
   "notification.messages.SEND_BILLING_RECEIPTS_ON_PAYDAY_REMINDER_MESSAGE.push.title": "Remind ☺️",
-  "notification.messages.MARKETPLACE_INVOICE_PUBLISHED_MESSAGE.push.title": "You have been invoiced for paid services",
-  "notification.messages.MARKETPLACE_INVOICE_WITH_TICKET_PUBLISHED_MESSAGE.push.title": "Pay the bill for paid services, ticket No. {ticketNumber}.️",
+  "notification.messages.MARKETPLACE_INVOICE_PUBLISHED_MESSAGE.push.title": "Your check \uD83D\uDD14",
+  "notification.messages.MARKETPLACE_INVOICE_WITH_TICKET_PUBLISHED_MESSAGE.push.title": "Your check \uD83D\uDD14️",
+  "notification.messages.MARKETPLACE_INVOICE_CASH_PUBLISHED_MESSAGE.push.title": "Your invoice for paid services \uD83D\uDD14",
```

Comment:
receipt is better:

a bill is a request for payment before it is made, a receipt is proof of payment issued after a transaction, and a check is a financial instrument used to transfer money from one party to another. Each serves a distinct role in the financial transaction process.

invoice is the same-ish as bill